Parse 0
195 1 ok
195 e[6]
[2022-09-22 12:54:44 run_p_internal]: startnodes=={0}, call_layer==0.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({0, 0}), pq.size()==1
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{1, 1}.
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{1, 2}.
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{1, 3}.
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 1}), pq.size()==4
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 2}), pq.size()==3
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 3}), pq.size()==2
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(3).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 1 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 2 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 3 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 4}), pq.size()==3
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 5}), pq.size()==2
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 4 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 5 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 0 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==1.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 1 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 2 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 4 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 5 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 1 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==2.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 1 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 2 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 4 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 5 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 2 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==3.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 1 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 2 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 4 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 5 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 3 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==4.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 1 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 2 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 4 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 5 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 4 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==5.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 1 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {1 (layer), 2 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 4 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_p_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {2 (layer), 5 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_p_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_p_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 4 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_p_internal]: The result of {3 (layer), 6 (idx), 4 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 3 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_p_internal]: The result of {3 (layer), 6 (idx), 3 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 2 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_p_internal]: The result of {3 (layer), 6 (idx), 2 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 1 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_p_internal]: The result of {3 (layer), 6 (idx), 1 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_p_internal]: Handle {3 (layer), 6 (idx), 0 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_p_internal]: The result of {3 (layer), 6 (idx), 0 (call_layer)} is not stored because result of layer5 has already been stored.

Parse 0
195 e[6]
[2022-09-22 12:54:44 run_p_internal]: startnodes=={0}, call_layer==0.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({0, 0}), pq.size()==1
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{1, 1}.
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{1, 2}.
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{1, 3}.
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 1}), pq.size()==4
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 2}), pq.size()==3
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 3}), pq.size()==2
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(3).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 1 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 2 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 3 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 4}), pq.size()==3
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 5}), pq.size()==2
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 4 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 5 (idx), 0 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 0 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==1.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 1 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 2 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 4 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 5 (idx), 1 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 1 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==2.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 1 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 2 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 4 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 5 (idx), 2 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 2 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==3.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 1 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 2 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 4 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 5 (idx), 3 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 3 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==4.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 1 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 2 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 4 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 5 (idx), 4 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line147:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 4 (call_layer)} failed because verdict==WRONG! Call 6 (retryudf) verdict==WRONG!
[2022-09-22 12:54:44 run_p_internal]: startnodes=={1, 2}, call_layer==5.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 1}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({1, 2}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(1).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(2).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 1 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 4}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {1 (layer), 2 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{2, 5}.
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 4}), pq.size()==2
E:\graph_simulator\engine\engine.h, line121:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({2, 5}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(4).
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(5).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 4 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line171:[2022-09-22 12:54:44 run_s_internal]: Push{3, 6}.
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {2 (layer), 5 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line115:[2022-09-22 12:54:44 run_s_internal]: samelayer.push_back({3, 6}), pq.size()==1
E:\graph_simulator\engine\engine.h, line129:[2022-09-22 12:54:44 run_s_internal]: launch(6).
E:\graph_simulator\engine\engine.h, line158:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 5 (call_layer)} successful. Verdict==2[Stored]
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 4 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_s_internal]: The result of {3 (layer), 6 (idx), 4 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 3 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_s_internal]: The result of {3 (layer), 6 (idx), 3 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 2 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_s_internal]: The result of {3 (layer), 6 (idx), 2 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 1 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_s_internal]: The result of {3 (layer), 6 (idx), 1 (call_layer)} is not stored because result of layer5 has already been stored.
E:\graph_simulator\engine\engine.h, line161:[2022-09-22 12:54:44 run_s_internal]: Handle {3 (layer), 6 (idx), 0 (call_layer)} unsuccessful (verdict==1).
E:\graph_simulator\engine\engine.h, line163:[2022-09-22 12:54:44 run_s_internal]: The result of {3 (layer), 6 (idx), 0 (call_layer)} is not stored because result of layer5 has already been stored.

